webapp/graphql/groups.js

```diff
@@ -171,8 +174,11 @@ export const groupQuery = (i18n) => {
           url
         }
         locationName
+        # ...location
         location {
```

Comment:
Fragment `location` should not have the same name as propety `location` I guess.
Sometime we have a GQL error seems related to this, see issue.